hatchery/nextflow.go

```diff
@@ -605,7 +605,7 @@ func ensureLaunchTemplate(ec2Svc *ec2.EC2, userName string, hostname string, job
 }
 
 // Create AWS Batch compute environment
```

Comment:
These look like just unused variables been there for several past commits? Great that you've got rid of these now. Do we have any lint for Golang that can catch these in the future?

Reply:
my local vscode golang linter caught them

the default golang linter enforces a lot of things but i guess that's not one of them. maybe it could be configured, from a quick search i'm not finding a quick solution and i don't have more time to spend on it, i don't think it's a major issue